computer_vision/brightness_augmentation.py

```diff
@@ -0,0 +1,88 @@
+import glob
+import os
+import random
+from string import ascii_lowercase, digits
+
+import cv2
+import numpy as np
+
+# Params
+IMAGE_DIR = ""
+OUTPUT_DIR = ""
+BRIGHTNESS_PERCENT = 50  # (between 0 and 100)
```

Comment:
Wouldn't it be better to add these directly to the functions?

Reply:
I think this way is easier for testing by the user.

Reply:
wouldn't it be better for the user to just reference the function. as most of the algorithms in this repository seems to a way for users to incorporate algorithms into their projects